README.md

```diff
@@ -11,3 +11,7 @@ and then `vagrant ssh`
 once in the terminal you can compile/run the project with
 
 `cargo run`
+
+# Tests
+
+`cargo test` to run the tests
```

Comment:
precise that the server must be up and running , as this part is different from other's crate `cargo test`